packages/omgx/wallet-frontend/src/deployment/local/tokens.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
with the latest develop, you can find the addresses from omgx/contracts/deployment

Reply:
We could copy that, but that will break `production`, since the copy step will not work. So getting things from http should be the default?

Reply:
In fact this entire token file should just be obtained from the `omgx_deployer`, correct?